### PR TITLE
Add fastq-pair v1.0 recipe

### PR DESCRIPTION
**Pull request by peterk87**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/fastq-pair/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/linsalrob/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/fastq-pair/meta.yaml]: please start with 0

File: recipes/fastq-pair/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+requirements:
+  build:
+    - cmake >=3.6
+    - {{ compiler('c') }}
+
+test:
+  files:
```
Comment (bgruening) [inline recipes/fastq-pair/meta.yaml]: We usually do not include test data in the repo. The command test should be enough.

If you like you can include a `make test` is it exists.

Comment (peterk87) [reply recipes/fastq-pair/meta.yaml]: Okay, thanks for the advice. I removed the test data and test script. Unfortunately, there's no `make test` for this tool.

File: recipes/fastq-pair/meta.yaml (hunk 1 above)
Comment (peterk87) [reply recipes/fastq-pair/meta.yaml]: Changed to 0

**Comment by druvus** [comment]
thanks @peterk87! fastq-pair was added in #13997

**Pull request closed without merging**